CONTRIBUTING.md

```diff
@@ -23,3 +23,33 @@ If you'd like to help make RSpec better, here are some ways you can contribute:
 If you need help getting started, check out the [DEVELOPMENT](DEVELOPMENT.md) file for steps that will get you up and running.
 
 Thanks for helping us make RSpec better!
+
+## Rspec issues labels definition
+
+### `Your first PR` issues
+
+These issues are the ones that we be believe are best suited for new
+contributors to get started on. They represent a potential meaningful
+contribution to the project that should not be too hard to pull off.
+
+### `Needs reproduction case` issues
+
+These issues are ones that have been labelled by the maintainers that we
+believe do not currently have enough information to be reproduced the RSpec
+team. While not directly counted by the GitHub contribution graph, we consider
+helping us to reproduced the issue with a repro case as an extremely meaningful
+contribution.
+
+### `Has reproduction case` issues
+
+Issues that have reproduction cases have a repository that we can clone that
```

Comment:
This head would be better to start by `These issues are the ones that...` because others start like that.

Reply:
Do you mean this part as follows?
```
These issues are the ones that have reproduction cases, able to start working on immediately. 
These are good ones to tackle to help us actively fix bugs.
```

Reply:
The former of your suggestions is good, I like that one